plugins/home/src/components/CustomHomepage/CustomHomepageButtons.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import Button from '@material-ui/core/Button';
+import React from 'react';
+import { createStyles, makeStyles, Theme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    responsiveGrid: {
+      '& .react-grid-item > .react-resizable-handle': {
+        mixBlendMode: 'difference',
+        borderRight: '2px solid rgba(255, 255, 255, 0.7)',
+        borderBottom: '2px solid rgba(255, 255, 255, 0.5)',
```

Comment:
Optional: Possibility to use `theme` palette colours? :)

Reply:
Good idea, just have to check what could fit here. I am not too happy the way resize corner looks anyway :D